src/flask_jsonrpc/types.py

```diff
@@ -46,7 +46,7 @@
 
 # Python 3.8+
 try:
-    from typing_extensions import Final
+    from typing import Final
```

Comment:
As this line is the same that inside the `except` block, please put this import on top of file (according to isort rules ;)